src/main/java/seedu/anilist/model/genre/Genre.java

```diff
@@ -59,8 +59,8 @@ public String toString() {
      * Compares the two genreNames, returns 0 if the other is null.
      */
     @Override
-    public int compareTo(Object o) {
-        if (!(o instanceof Genre)) {
+    public int compareTo(Genre o) {
+        if (o == null) {
             return 0;
         }
         Genre g = (Genre) o;
```

Comment:
Then in that case should line 66 also be removed? Since o is already a genre

Reply:
oh yah